lib/riemann/tools/health.rb

```diff
@@ -23,7 +23,9 @@ class Health
       opt :memory_critical, 'Memory critical threshold (fraction of RAM)', default: 0.95
       opt :uptime_warning, 'Uptime warning threshold', default: 86_400
       opt :uptime_critical, 'Uptime critical threshold', default: 3600
-      opt :checks, 'A list of checks to run.', type: :strings, default: %w[cpu load memory disk]
+      opt :swap_warning, 'Swap warning threshold', default: 0.4
+      opt :swap_critical, 'Swap critical threshold', default: 0.5
```

Comment:
I don't think there are one-size-fit-all thresholds: I chose these ones based on old way of thinking that you configure twice the amount of RAM as swap: half of it is"reserved" for a dump of the memory in case of crash (so critical threshold is 0.5), and arbitrarily using 80% of the remaining is a warning threshold :shrug:.

Reply:
Ah. Yeah makes sense.